s31052-LukaszBurnicki/zajecia3/zadanie1

```diff
@@ -0,0 +1,37 @@
+Program ps służy do wyświetlania listy procesów. 
+
+Zadania:
+
+A: Zobacz co się stanie jeśli wpiszemy w terminalu:
+
+    ps
+	wyswietla aktualne procesy
+
+    ps -a
+	wyswietla procesy z wyjatkiem liderow i niezwiazanych z terminalem
+
+    ps x
+	wyswietla wszystkie aktualne procesy,
+	te z terminalem i nie związane z nim
+
+    ps axu
+	wyswietla procesy wykonywane na koncie wszystkich zalogowanych
+	uzytkownikow oraz te niezwiazane z terminalem
+
+    INFO: tu kolejność parametrów a, x, u nie ma znaczenie a jedynie fakt,
+	że są podane. Dowolna kolejność powinna dać ten sam wynik.
+
+    Jak myślisz, co oznacza znak zapytania w kolumnie 2?
+	nie ma terminala sterującego
+
+    Nie wiesz? A gdzie może być wyjaśnienie?
+	znalazłem w internecie, od razu wyskoczyło mi po zadaniu pytania
+	co oznacza "?" w kolumnie TTY
+
+B: Jak wykonasz poniższe 2 zadania?
+
+    Wyświetl wszystkie procesy bash
+	ps aux | grep bash
```

Comment:
Jeśli będzie w systemie użytkownik o loginie `barnabash` i będzie miał uruchomiony jakiś inny proces niż bash to mimo wszystko także zostanie wypisany